.ci_support/build_notebooks.sh

```
@@ -0,0 +1,10 @@
+# execute notebooks
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)